app/Http/Controllers/ProjectController.php

```diff
@@ -26,25 +26,29 @@ public function show($id) {
 
 	public function create() {
 		return view(
-			'registerProject', [
+			'projects.create', [
 				'courses' => Auth::user()->EnrolledIn,
+				'teams' => Auth::user()->teamsLed()->get(),
 				'labels' => Tag::where('type', 2)->get(),
 				'skillsets' => Tag::where('type', 1)->get()
 			]
 		);
 	}
 
 	public function store(Request $request) {
+
 		$attributes = request()->validate([
-			'projectName' => ['required'],
-			'videoPitch' => ['required'],
-			'longDescription' => ['required'],
-			'shortDescription' => ['required'],
-			'courses' => ['required'],
-			'projectImage' => ['required'],
-			'label' => 'required|array|min:1',
+			'projectName' => ['required', 'min:3'],
+			'videoPitch' => 'required',
+			'longDescription' => 'required',
+			'shortDescription' => 'required',
+			'course' => 'required',
```

Comment:
Should `course` be required? 

Sé que en el flujo de abajo está definido así pero me estoy preguntando si eso está bien porque creo que queríamos la opción de desasociar un proyecto de un curso y entonces no tiene mucho sentido para mí que lo tengamos que forzar a asociarlo al momento de crearlo.   

![image](https://user-images.githubusercontent.com/14140121/55037919-df5c0b80-4fe4-11e9-9965-0a332e92e26b.png)

Reply:
Sí, aquí sí porque eso es un feature que nosotros no contemplamos agregar ahorita y en tanto no lo agreguemos deberíamos forzarlo.